### DISCUSSION TITLE
How to use QuickVC?

### DISCUSSION THREAD

**Discussion by FPGithubber** [voicepaw/so-vits-svc-fork#249] [Q&A] [answered]
I wanted to try QuickVC, so I used the following setting: 
"svc pre-config -t quickvc" 

I started the training, and after a while I wanted to hear the first result... I opened the gui, selected the model, the config, and the input file - but the result was only an electronic "noise" after pressing infer. (Wav is attached)

Is there any option that needs to be adjusted? Is there anything that needs to be taken into account?
Can someone please help me to get it right? 

And for that matter, could anyone please explain what the difference is between the normal method and the use of QuickVC?
Are there any advantages? Is it faster? Is the quality better in comparison?

Thanks a lot.

[untitled.out.wav.zip](https://github.com/34j/so-vits-svc-fork/files/11182616/untitled.out.wav.zip)

**Comment by 34j** [comment, marked as answer]
Please see #242
